pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -293,32 +147,58 @@ func (cs *ConnectionStats) updateStatsWorker() {
 	defer tk.Stop()
 
 	for {
-		select {
-		case <-cs.done:
+		<-tk.C
```

Comment:
this is super minor but we should standardize on using selects. i'll fix the place i did this in room the other day. there's no reason to wait for the next tick to exit these loops.

Reply:
That would be great to not wait. Please let me know how to do that.

Reply:
the way you had it before was good
```go
select {
case <-cs.done:
    return
case <-ticker.C:
    // work...
}
```

Reply:
cool, will change it back. Was trying to get rid of one field in the struct 😄

Reply:
oh, we could use dc's fuse if you want something more ergonomic

Reply:
@paulwe Changed in this [commit](https://github.com/livekit/livekit/pull/1490/commits/45a420fa06ae35c6ce7ef568f2dd74b43a9fa0c2). Please check when you get a chance.

Reply:
lgtm 👍